pom.xml

```diff
@@ -19,40 +19,31 @@
         <artifactId>jberet-parent</artifactId>
         <version>2.1.1.Final</version>
     </parent>
-
     <artifactId>jberet-jpa-repository</artifactId>
+    <version>2.1.1.Final-SNAPSHOT</version>
 
     <properties>
-        <version.org.hibernate>5.6.2.Final</version.org.hibernate>
+        <version.jakarta.persistence>3.1.0</version.jakarta.persistence>
+        <version.org.hibernate>6.1.1.Final</version.org.hibernate>
+        <version.junit>4.13.2</version.junit>
```

Comment:
do we really need to override this junit version? The parent pom has 4.13.1.

Reply:
you are right, 4.13.1 is just fine